client/app/home/home.html

```diff
@@ -1,8 +1,11 @@
 <center>
+    <div class="row spacer" style="margin-top:50px"></div>
     <div>
-        <img id="spinit" src="assets/images/headerlogofront.png" width="300"/>
+        <!-- <img id="spinit" src="assets/images/headerlogofront.png" width="300"/> -->
+        <p style="font-family: Kanit; font-size:200px"><i>OPENEXP</i></p>
```

Comment:
good question.  In the logo, I think it might look best in all caps.  In writing, I like it as OpenEXP.  Does that seem reasonable or confusing? haha